src/main/scala/nl.knaw.dans.easy.bagstore/server/StoresServletComponent.scala

```diff
@@ -147,21 +147,28 @@ trait StoresServletComponent extends DebugEnhancedLogging {
       trace(())
       basicAuth()
       debug("Authenticated")
-      val bagstore = params("bagstore")
+      val bagStore = params("bagstore")
       val uuidStr = params("uuid")
-      logger.info(s"Received PUT request for bagstore $bagstore and UUID $uuidStr")
-      bagStores.getBaseDirByShortname(bagstore)
+      val contentType = Option(request.getHeader("Content-Type"))
```

Comment:
can you rename `contentType` to `requestContentType`? the former is a method name in Scalatra...